### ISSUE TITLE
Google Drive "open in: Photopea" and problem with editing smart objects

### ISSUE THREAD

**Issue by MichalSZZ** [photopea/photopea#1201]
My file: https://drive.google.com/open?id=1lifeBmIDCIkMMnSWHBXT52peOceH5XJF
When I try to edit smart object (double click on layer thumbnail). New tab is opening but, layers work/look incorrect.

Please check these layers: 
-  "exam-laptop-top-view"
-  and any of "podkladka" layer

Download file and check ones again these layers how they look like when edit smart object

**Comment by MichalSZZ** [comment]
<img width="1160" alt="obraz" src="https://user-images.githubusercontent.com/38118273/51793040-b5d85c80-21ba-11e9-8a35-9d532bd997f3.png">
Editing smart object online stored on Google Drive 
<img width="1160" alt="obraz" src="https://user-images.githubusercontent.com/38118273/51793070-02bc3300-21bb-11e9-85ea-db35343b0a14.png">
Editing smart object saved on HDD

Another smart object in this file:
<img width="1160" alt="obraz" src="https://user-images.githubusercontent.com/38118273/51793086-42831a80-21bb-11e9-93f8-b848fdd8b68b.png">
vs
<img width="1156" alt="obraz" src="https://user-images.githubusercontent.com/38118273/51793097-5fb7e900-21bb-11e9-9d49-350d64a209f8.png">
in local editing

**Comment by photopea** [comment]
Hi, I uploaded your PSD to Google Drive and opened it from there, and opened a Smart Object. It looks like your version of Local editing - HDD.

When you open from Google Drive, can you save it locally as PSD, and then, open that PSD locally?

**Comment by MichalSZZ** [comment]
Ok I saved from Google Drive and opened and looks good.

Now I see that, uncheck the "Use webGL" helps to me. But that bug is only in "online" editing of smart object (the same file).

**Comment by photopea** [comment]
Your two cases looked different because of different environments (different computer / browser etc). It is not related to Google Drive at all. One environment seems to have a bug (the one that looks incorrect).

So the image looks correctly when WebGL is enabled or disabled? Can you update the browser / OS on that device?

**Comment by MichalSZZ** [comment]
I just updated my Firefox to newest one 64.0.2 on my Mac OS (High Sierra).
Same problem. I have "Use webGL" checked when bug is visible.

But as you have described - now I see that is not the Google Drive problem :( Sorry... I should check that before. Stupid mistake - when I opened that file localy I have used PWA version with Chrome engine. That was the problem - some bug in FF or in my OS/drivers.
